### ISSUE TITLE
Write a document describing what the tool does and _does not_ do vs Accessibility

### ISSUE THREAD

**Issue by bdeclerc** [openfed/AccessibilityCheck#3]
We'll need to write, next to the Readme, a more elaborate document describing the relationship of this tool vs the WCAG and EU guidelines, since it inherently cannot cover 100% of the accessibility rules (for example, it can only validate if there's a filled in alt-tag on an image, not if the content of that alt-tag is an accurate description of the image, same for form-labels.

**Comment by bdeclerc** [comment]
@stefanruijsenaars note: this is likely a task we will outsource to someone else, your talents are better used in developing the tools than in documenting accessibility guidelines :)

**Comment by bdeclerc** [comment]
https://openfed.github.io/AccessibilityCheck/overview/
